### ISSUE TITLE
CasiTestati vs Tamponi

### ISSUE THREAD

**Issue by contenitore21** [pcm-dpc/COVID-19#594]
qual è il significato dell'ultima colonna apparsa ieri CasiTestati rispetto al numero tamponi? e come mai varia da 100% della basilicata al 25% del lazio? grazie

**Comment by vi-enne** [comment]
Numero tamponi si riferisce al numero di tamponi effettuati, mentre casi testati si riferisce al numero di singole persone sottoposte a tampone.

Dato che una stessa persona può essere sottoposta a più di un tampone (ad esempio, per accertarne al guarigione), numero tamponi >= numero casi testati.

Per alcune regioni, questi due numeri sono uguali. Tre spiegazioni possibili:
- o indicano i dimessi, cioè le persone positive senza più sintomi, cosi facendo un solo tampone a persona;
- o il numero dei tamponi effettuati comunicato dall'inizio è in realtà quello di persone testate;
- o c'è un errore.

**Comment by Patrizia1972** [comment]
Scusate ma per il Lazio risultano a oggi un totale di 86545 casi testati su 114317 tamponi totali... ieri 26298 casi testati su 111073 tamponi... le differenze tra un giorno e l'altro che mi danno i nuovi totali della sola giornata non sono plausibili. Mi risultano 60247 (86545 -26298 ) casi testati su 3244 (114317 -111073 ) tamponi. Come è possibile? Grazie mille

**Comment by contenitore21** [comment]
evidentemente i dati del lazio di ieri erano sballati, anche perchè 4 tamponi a persona sembravano tanti

**Comment by Patrizia1972** [comment]
Ma io li ho riscaricati tutti di nuovo... ieri c'era l'errore che i dati non erano ordinati in modo corretto, quindi oggi ho ripreso quelli totali ... ma il dato del Lazio risulta quello che ho indicato... non so se è solo il caso del Lazio

**Comment by 123cobloc** [comment]
c'è una nota che indica che i dati del Lazio di ieri sono parziali

**Comment by kak93** [comment]
Buonasera, ho notato che il numero di casi testati giornalieri di oggi è superiore al numero di tamponi effettuati e questo ovviamente non può essere perchè il numero di casi testati puo al massimo essere pari al numero dei tamponi. Sicuramente hanno aggiunto i testati mancanti tutti insieme oggi e questo scombina tutto. Invece di ridistribuirli dove mancavano nei giorni precedenti. Stavo portando avanti una curva di contagio normalizzata sul numero delle persone testate ma ora questa sarà falsata dal numero errato di casi testati.

**Comment by Patrizia1972** [comment]
Infatti, probabilmente è quello che è successo nel Lazio e non so se in altre regioni. Peccato perchè così i dati non sono utilizzabili... speriamo si possano correggere

**Comment by kak93** [comment]
peccato se qualcuno nei prossimi giorni nota delle correzioni lo faccia presente.

**Comment by ChiaraCCC** [comment]
Il problema si trova anche sui dati aggregati. Il 23 Aprile si riportano 1.579.90 tamponi effettuati, mentre il 24 Aprile i tamponi sono 1.642.35, con una differenza di **62.447 tamponi nuovi**.
Tuttavia i casi testati il 23 e 24 Aprile sono, rispettivamente, 1.052.577 e 1.147.850, quindi ci sono **95.273 nuovi casi.**

Palesemente ci deve essere un errore, visto che ci sono circa 30.000 persone che sono state testate senza tampone? Non ha alcun senso.

**Comment by kak93** [comment]
È esattamente quello che intendevo infatti. Ci sta una disparità tra i tamponi fatti oggi e i casi testati oggi. Sono maggiori i casi testati, cosa ovviamente impossibile.

**Comment by Murray1991** [comment]
> evidentemente i dati del lazio di ieri erano sballati, anche perchè 4 tamponi a persona sembravano tanti

Se consideri che per persona contagiata hai 1 tampone per verificarne la positività e almeno 2 per verificarne che è guarita, siamo già a 3. Capita comunque che tu possa avere più di 2 tamponi per verificare la guarigione della persona

**Comment by contenitore21** [comment]
> > evidentemente i dati del lazio di ieri erano sballati, anche perchè 4 tamponi a persona sembravano tanti
> 
> Se consideri che per persona contagiata hai 1 tampone per verificarne la positività e almeno 2 per verificarne che è guarita, siamo già a 3. Capita comunque che tu possa avere più di 2 tamponi per verificare la guarigione della persona

giusto, ma allora sono sballati quelle di altre regioni dove si arriva financo a un rapporto 1 a 1. C'è troppa disomogeneità tra i rapporti Tamponi/casi per pensare che il dato sia trattato allo stesso modo. E a questo punto anche il rapporto nuovi positivi/tamponi (o casi) è di difficile interpretazione.

**Comment by RiccardoBorchi** [comment]
Scusate, ma questo problema non è ancora stato risolto. Il 24 Aprile risultano essere state testate **95.273** persone, ma il numero dei tamponi lo stesso giorno è di **62.447**, il che è impossibile, come è già stato fatto notare.

@umbros potreste correggere questo dato?

Grazie.

**Comment by RiccardoBorchi** [comment]
@umbros riuscite a fare questo aggiornamento?

Grazie.
